main.go

```diff
@@ -58,11 +60,15 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
+	var (
+		metricsAddr string
+		probeAddr   string
+		clusterName string
+	)
 	var enableLeaderElection bool
-	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.StringVar(&clusterName, "cluster-name", "kubegres", "The name of the cluster. This is used to identify the cluster in case of multi-cluster deployments.")
```

Comment:
Isn't each kubegres spec a custer itself?
Is this one referring to the k8s cluster name? Should we use better naming?

What about:
- k8s-cluster-name
- k8s-environment

I know this affects part of the code used to generate the replication slot name.